classes/class-wc-connect-api-client.php

```diff
@@ -295,6 +295,30 @@ public function auth_test() {
 			return $this->request( 'GET', '/connection/test' );
 		}
 
+		/** Heartbet test
+		 * 
+		 * @return true|WP_Error
+		 */
+		public function is_alive() {
+			return $this->request( 'GET', '' );
+		}
+
+		/** Heartbet test with transient cache.
+		 * 
+		 * @return true|WP_Error
+		 */
+		public function is_alive_cached() {
+			$connect_server_is_alive_transient = get_transient( 'connect_server_is_alive_transient' );
+			if ( true === $connect_server_is_alive_transient ) {
```

Comment:
This check is not working for me because get_transient is returning `"1"` for true. I think a `false !== $connect_server_is_alive_transient` should work because get_transient returns the bool false for expired transients.

Reply:
Corrected.